editoast/src/modelsv2/study.rs

```diff
@@ -85,74 +85,71 @@ fn dates_in_order(a: Option<Option<NaiveDate>>, b: Option<Option<NaiveDate>>) ->
 pub mod test {
     use rstest::rstest;
     use std::ops::DerefMut;
-    use std::sync::Arc;
 
     use super::*;
-    use crate::fixtures::tests::db_pool;
-    use crate::fixtures::tests::study_fixture_set;
-    use crate::fixtures::tests::StudyFixtureSet;
-    use crate::fixtures::tests::TestFixture;
-    use crate::modelsv2::DbConnectionPool;
+    use crate::modelsv2::fixtures::create_project;
+    use crate::modelsv2::fixtures::create_study;
+    use crate::modelsv2::DbConnectionPoolV2;
 
     #[rstest]
-    async fn create_delete_study(
-        #[future] study_fixture_set: StudyFixtureSet,
-        db_pool: Arc<DbConnectionPool>,
-    ) {
-        let StudyFixtureSet {
-            study,
-            project: _project,
-        } = study_fixture_set.await;
-
-        // Delete the study
-        let conn = &mut db_pool.get().await.unwrap();
-        Study::delete_static(conn, study.id()).await.unwrap();
-
-        // Second delete should fail
-        assert!(!Study::delete_static(conn, study.id()).await.unwrap());
+    async fn study_creation() {
```

Comment:
This test is already included into `study_retrieve`. I'd say it's fine to remove it.